test/development/acceptance/ReactRefreshLogBox-scss.test.ts

```diff
@@ -3,7 +3,7 @@ import { sandbox } from './helpers'
 import { createNext } from 'e2e-utils'
 import { NextInstance } from 'test/lib/next-modes/base'
 
-describe('ReactRefreshLogBox', () => {
+describe.skip('ReactRefreshLogBox', () => {
```

Comment:
Should we add a comment here why it was skipped?

Reply:
Added in https://github.com/vercel/next.js/pull/29264/commits/0d3e7c40ed17dc1c4e25064dae25df18ef9e666c